client/scraper/youtube_metadata.py

```diff
@@ -32,7 +32,7 @@ def parse_content(self, response):
         media_list = {}
         media_list[FieldKeyword.DATA] = [
             {
-                FieldKeyword.URL: None,
+                FieldKeyword.URL: response.request_url,
```

Comment:
What's the reasoning behind this living inside the media_list rather than outside as part of the prop_map?